vars/buildImage.groovy

```diff
@@ -92,9 +92,9 @@ def makeImageStep(String pipeline_version, String arch, String debian_arch, Stri
                 }
 
                 stage("Upload images for ${arch}") {
-                    withCredentials([string(credentialsId: 'Staging KernelCI API Token', variable: 'API_TOKEN')]) {
+                    withCredentials([string(credentialsId: 'KernelCI Upload API Token', variable: 'API_TOKEN')]) {
                         sh """
-                            python push-source.py --token ${API_TOKEN} --api https://staging-api.kernelci.org \
```

Comment:
Same thing for the API URL, there's a job parameter for that in build and bisection jobs:

https://github.com/kernelci/kernelci-build/blob/master/jenkins/build.jpl#L101
```
API=${params.KCI_API_URL} \
TOKEN=${SECRET} \
${kci_build}/build.py -i ${opt} -c ${defconfig}
```